gapic/src/main/com/google/gapid/perfetto/views/RootPanel.java

```diff
@@ -212,6 +212,10 @@ public Dragger onDragStart(double sx, double sy, int mods) {
       return Dragger.NONE;
     }
 
+    if (mouseMode == MouseMode.Flagging) {
```

Comment:
We shouldn't do an early return here, just let returning Dragger.NONE happen at Line 237. 
Reason: the next several lines will handle mode overriding: basically we should let [shift-dragging -> selection; ctrl-dragging -> time higlight; space-dragging- > pan] to work in any modes. Early return here will disable the mode overriding in flagging mode.

All other code looks fine for me now, just this one last bit introduced in the new commit.

Reply:
Yes I intentionally made the override in flagging mode. Because I accidentally went into timing select when flagging because of misclicks and that started becoming a bit messy. But we can change this behavior after we finalize something for the UX.